lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -70,11 +70,19 @@ foreach ($member in $members) {
     }
 
     if ($state -in @("present", "pure") -and !$user_in_group) {
-        Add-ADGroupMember -Identity $name -Members $group_member -WhatIf:$check_mode @extra_args
+        try {
+            Add-ADGroupMember -Identity $name -Members $group_member -WhatIf:$check_mode @extra_args
+        } catch {
+            Fail-Json $result "Failed to add a group $($group_member): $($_.Exception.Message)"
```

Comment:
A silly comment but probably should be `Failed to add group $($group_member): $($_.Exception.Message)` or `Failed to add a group: $($group_member) - $($_.Exception.Message)`